sound/soc/sof/ipc.c

```diff
@@ -268,11 +268,10 @@ int sof_ipc_tx_message(struct snd_sof_ipc *ipc, u32 header,
 
 	spin_unlock_irq(&sdev->ipc_lock);
 
-	/* schedule the message if not busy */
-	if (!sdev->disable_ipc_queue && snd_sof_dsp_is_ipc_ready(sdev))
-		ret = ipc_tx_next_msg(sdev->ipc);
-	else
```

Comment:
could this be squashed/handled differently? The change above was added a couple of patches ago and is again modified here.

Reply:
I agree, that some patches can and should be squashed together, like 01/11 and 11/11, but in this case I think those are two different changes that just happen to touch the same code.